app-benchmarks/bonnie++/bonnie++-1.97-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+inherit eutils
```

Comment:
ditch `eutils`, just use `PATCHES` array and ensure all patches apply with `-p1`